package.json

```diff
@@ -110,12 +110,14 @@
     "gulp-connect": "3.2.2",
     "html-loader": "0.4.3",
     "html-webpack-plugin": "2.16.0",
+    "http-server": "github:johntron/http-server#proxy-secure-flag",
```

Comment:
I have seen that happening on my previous employer. It takes a while to debug that the CI goes boom because of that 😥 

There we decided to have our own fork.

Reply:
eventually we will have the mockserver… :)

Reply:
Couldn't we just use http-server on it's own?

Reply:
hm, yes, but now in this PR. we are using that branch all over the place :(

Reply:
@juliangieseke would you please create a jira for forking http-server?

Reply:
👍  https://jira.mesosphere.com/browse/DCOS-22381